package-testing/pre/000_install_package.rb

```diff
@@ -2,13 +2,24 @@
   workstation = find_at_most_one('workstation')
 
   step 'Install pdk package' do
+    if ENV['LOCAL_PKG']
+      pkg = File.basename(ENV['LOCAL_PKG'])
+      scp_to(workstation, ENV['LOCAL_PKG'], pkg)
```

Comment:
It's worth testing the local file exists before calling scp_to because beaker doesn't have a lot of parameter validation and probably won't cleanly fail if the local file doesn't exist.
Though we might be already covered by the call to File.basename on L6